Mathlib/Topology/Algebra/Valued/ValuativeRel.lean

```diff
@@ -35,21 +35,54 @@ open ValuativeRel TopologicalSpace Filter Topology Set
 
 local notation "v" => valuation R
 
+lemma mem_nhds_iff (s : Set R) : s ∈ 𝓝 (0 : R) ↔
+    ∃ γ : (ValueGroupWithZero R)ˣ, { x | v x < γ } ⊆ s := by
+  convert IsValuativeTopology.mem_nhds (x := (0 : R))
+  rw [sub_zero]
+
+theorem hasBasis_nhds (x : R) :
+    (𝓝 (x : R)).HasBasis (fun _ => True)
+      fun γ : (ValueGroupWithZero R)ˣ => { z | v (z - x) < γ } := by
```

Comment:
```suggestion
    (𝓝 x).HasBasis (fun _ => True)
      fun γ : (ValueGroupWithZero R)ˣ => { z | v (z - x) < γ } := by
```
And the following version would also be useful.
```suggestion
    (𝓝 x).HasBasis (· ≠ 0) ({ z | v (z - x) < · } := by
```

Reply:
Maybe this would belong better to #27163 instead?